msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2021-05-25 15:06+0000\n"
"Last-Translator: \n"
"Language-Team: Norwegian Bokmal\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Project-ID: 289505\n"
"X-Crowdin-Language: nb\n"
"X-Crowdin-File-ID: 670\n"
"Language: nb_NO\n"

#: redbot/cogs/audio/core/events/cog.py:215
msgid "Auto Play started."
msgstr "Automatisk avspilling startet."

#: redbot/cogs/audio/core/events/dpy.py:40
msgid "You have attempted to run Audio's Lavalink server on an unsupported architecture. Only settings related commands will be available."
msgstr "Du har forsøkt å kjøre Audios Lavalink server på en konstruksjon som ikke støttes. Bare relaterte kommandoer vil være tilgjengelig."

#: redbot/cogs/audio/core/events/dpy.py:70
msgid "I'm missing permissions in this server, Please address this as soon as possible.\n\n"
"Expected Permissions:\n"
msgstr "Jeg mangler tillatelser i denne serveren, Vennligst adresse dette så snart som mulig.\n\n"
"Forventede tillatelser:\n"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Enabled"
msgstr "Aktivert"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Disabled"
msgstr "Deaktivert"

#: redbot/cogs/audio/core/events/dpy.py:124
msgid "No DJ role found. Disabling DJ mode."
msgstr "Ingen DJ rolle funnet. Deaktivering av DJ modus."

#: redbot/cogs/audio/core/events/dpy.py:134
msgid "`{user_input}` is not a valid value for `{command}`"
msgstr "`{user_input}` er ikke en gyldig verdi for `{command}`"

#: redbot/cogs/audio/core/events/dpy.py:142
msgid "Unable To Parse Argument"
msgstr "Kan ikke analysere argument"

#: redbot/cogs/audio/core/events/dpy.py:154
#: redbot/cogs/audio/core/events/dpy.py:163
msgid "Invalid Argument"
msgstr "Ugyldig Argument"

#: redbot/cogs/audio/core/events/dpy.py:155
msgid "The argument you gave for `{}` is not valid: I was expecting a `{}`."
msgstr "Argumentet du ga for `{}` er ikke gyldig: Jeg forventet en `{}`."

#: redbot/cogs/audio/core/events/dpy.py:175
msgid "Invalid Environment"
msgstr "Ugyldig miljø"

#: redbot/cogs/audio/core/events/dpy.py:176
msgid "Connection to Lavalink has been lost."
msgstr "Forbindelsen til Lavalink har gått tapt."

#: redbot/cogs/audio/core/events/dpy.py:184
msgid "No Player Available"
msgstr "Ingen spiller tilgjengelig"

#: redbot/cogs/audio/core/events/dpy.py:185
msgid "The bot is not connected to a voice channel."
msgstr "Botten er ikke koblet til en talekanal."

#: redbot/cogs/audio/core/events/dpy.py:193
#: redbot/cogs/audio/core/events/lavalink.py:162
msgid "Unable to Get Track"
msgstr "Kunne ikke motta spor"

#: redbot/cogs/audio/core/events/dpy.py:194
#: redbot/cogs/audio/core/events/lavalink.py:163
msgid "I'm unable to get a track from Lavalink at the moment, try again in a few minutes."
msgstr "Jeg klarer ikke å finne et spor fra Lavalink i øyeblikket, prøv igjen om noen minutter."

#: redbot/cogs/audio/core/events/dpy.py:205
msgid "There was an issue communicating with Discord."
msgstr "Det var et problem som kommuniserte med Discord."

#: redbot/cogs/audio/core/events/dpy.py:206
msgid "This error has been reported to the bot owner."
msgstr "Denne feilen er rapportert til bot eieren."

#: redbot/cogs/audio/core/events/lavalink.py:154
msgid "Couldn't get a valid track."
msgstr "Kunne ikke få et gyldig spor."

#: redbot/cogs/audio/core/events/lavalink.py:189
msgid "Now Playing"
msgstr "Spiller nå"

#: redbot/cogs/audio/core/events/lavalink.py:191
msgid "Track length: {length} | Requested by: {user}"
msgstr "Sporlengde: {length} | Forespurt av: {user}"

#: redbot/cogs/audio/core/events/lavalink.py:212
msgid "Queue ended."
msgstr "Kø avsluttet."

#: redbot/cogs/audio/core/events/lavalink.py:264
msgid "Multiple Errors Detected"
msgstr "Flere feil oppdaget"

#: redbot/cogs/audio/core/events/lavalink.py:265
msgid "Closing the audio player due to multiple errors being detected. If this persists, please inform the bot owner as the Audio cog may be temporally unavailable."
msgstr "Hvis du lukker lydspilleren på grunn av flere feil blir oppdaget. Hvis dette vedvarer, må du informere bot eieren om at lydloggen kan være midlertidig utilgjengelig."

#: redbot/cogs/audio/core/events/lavalink.py:279
msgid "Track Stuck"
msgstr "Spor satt fast"

#: redbot/cogs/audio/core/events/lavalink.py:280
msgid "Playback of the song has stopped due to an unexpected error.\n"
"{error}"
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:286
msgid "Track Error"
msgstr "Sporfeil"

